runtime/doc/news.txt

```diff
@@ -13,6 +13,23 @@ BREAKING CHANGES                                                *news-breaking*
 
 The following changes may require adaptations in user config or plugins.
 
+Cscope (and all its relations) were removed:
```

Comment:
Do we want to add a link to the PR to these items (and I do think they should be itemized)?

Reply:
This reads a little ambiguous to me. A link from where to where?

Reply:
I meant a link to this PR so people can read up on details and discussions, if interested. Something like 

```
- [#20545] Cscope (and all its relations) were removed:
...
```

Reply:
(The first PR that adds an entry -- either yours or my filetype.vim excision -- sets the format for later PRs, so this should be discussed now.)

Reply:
> If people want a PR/issue link, they can just look at the git history which is where all of this information is supposed to live.

I think that is a bit glib; experience has shown that it's not trivial for everyone to dig this information up.

I do agree that littering the news file with hyperlinks is not ideal; on the other hand I expect that many (most?) users will consume this via https://neovim.io/doc/user/news.html, where these links could be unobtrusive.

(I wish vimdoc would support proper hyperlinks like markdown; then we could move the actual links out of the way at the bottom of the file.)

Reply:
> I think that is a bit glib; experience has shown that it's not trivial for everyone to dig this information up.

Maybe that has a silver lining and that it shouldn't be trivial. I'm not sure if we want a whole herd of users commenting on our tracker, we've seen many times what happens when Mr. casual gives their 2 cents. I'd rather let reddit do that.

I don't know about glib, but our git history does contain PR refs so the only requirement from a user to find out more information is to have the code checked out and run `git log`.

Reply:
> so the only requirement from a user to find out more information is to have the code checked out and run git log.

Yes, and I'm saying that I think this is a bit too much friction, especially since most users install via package managers and not from source. It's also a non-trivial inverse problem to find the right commit given only a high-level description of the change.

Reply:
Also,

> So far we have avoided anything like issue refs in the help files haven't we?

So far we haven't had anything like this kind of information in the help files; hence we need to discuss this now rather than refer to a previous decision.

Reply:
Hmm, ok. Maybe for other changes, such a link would be useful. For this change there really isn't any discussion (except for #14964) since most of it was done in matrix.

Reply:
As I said, this is a general question that only hits you because you are the first to add an item. We can take this offline, if you prefer.

Reply:
True. It's just easier to remove unnecessary information than to dig up and add it ;)

Reply:
* In the same breath we should mention that `ctags will NEVER be removed, only cscope was removed.`
* Instead of listing all these things here we could link to `:help cscope` or `:help nvim-features-removed` ?

Reply:
> In the same breath we should mention that ctags will NEVER be removed, only cscope was removed.

"never" is very strong, but I'll mention there are no plans to remove it.

> Instead of listing all these things here we could link to :help cscope or :help nvim-features-removed ?

The items here are a bit scattered in `vim_diff.txt`, so I think it's there's value in having them grouped here.